RevenueCatUI/CustomerCenter/Views/FeedbackSurveyView.swift

```diff
@@ -44,11 +44,13 @@ struct FeedbackSurveyView: View {
 
     init(
         feedbackSurveyData: FeedbackSurveyData,
+        purchasesProvider: CustomerCenterPurchasesType = CustomerCenterPurchases(),
```

Comment:
Where do we draw the line now? I mean, I see that in some places the default init has been removed 🤔

Reply:
Yes, you're right I've not been consistent on this